src/transformers/trainer.py

```diff
@@ -1253,18 +1253,36 @@ def torch_jit_model_eval(self, model, dataloader, training=False):
                 return model
             jit_inputs = []
             example_batch = next(iter(dataloader))
+            if "labels" in example_batch:
+                if self.label_smoother is not None:
+                    example_batch.pop("labels")
+                elif version.parse(version.parse(torch.__version__).base_version) < version.parse("1.14.0"):
+                    logger.warning(
+                        "fail to use PyTorch jit mode due to trace could not handle `labels` input. you could use"
+                        " --label_smoothing_factor 0.1 for WA"
+                    )
+                    return model
             for key in example_batch:
                 example_tensor = torch.ones_like(example_batch[key])
                 jit_inputs.append(example_tensor)
             jit_inputs = tuple(jit_inputs)
+
             try:
                 jit_model = model.eval()
-                with ContextManagers([self.autocast_smart_context_manager(), torch.no_grad()]):
+                with ContextManagers(
+                    [
+                        torch.cpu.amp.autocast(cache_enabled=False, dtype=self.amp_dtype)
+                        if self.use_cpu_amp
+                        else contextlib.nullcontext(),
+                        torch.no_grad(),
```

Comment:
Why are you removing support for GPU mixed precision here?

Reply:
thanks for the reminding, add it back. and verified in GPU